### PR TITLE
Added caveat for version-specific patches

### PR DESCRIPTION
**Pull request by jtravee**
- Adding note that patches are needed for only 2.6.5, 2.6.6

### CODE REVIEW

**Review by btat — APPROVED** [review]

File: content/rancher/v2.6/en/neuvector-integration/_index.md [hunk 1]
```diff
@@ -108,6 +108,9 @@ Below are the minimum recommended computing resources for the NeuVector chart in
 
 - NeuVector components Controller and Enforcer are deployable if PSP is set to true.
 
+
+**Applicable to Rancher v2.6.5 and v2.6.6 only:**
```
Comment (anupama2501) [inline content/rancher/v2.6/en/neuvector-integration/_index.md:112]: Should be changed to Applicable to NeuVector chart version 100.0.0+up2.2.0

File: content/rancher/v2.6/en/neuvector-integration/_index.md [hunk 2]
```diff
@@ -126,6 +129,9 @@ To enable SELinux on RKE2 clusters, follow the steps below:
...
+
```
Comment (anupama2501) [inline content/rancher/v2.6/en/neuvector-integration/_index.md:134]: Should be changed to Applicable to NeuVector chart version 100.0.0+up2.2.0

File: content/rancher/v2.6/en/neuvector-integration/_index.md (hunk 1 above)
Comment (jtravee) [reply content/rancher/v2.6/en/neuvector-integration/_index.md]: Updated as recommended.

File: content/rancher/v2.6/en/neuvector-integration/_index.md (hunk 2 above)
Comment (jtravee) [reply content/rancher/v2.6/en/neuvector-integration/_index.md:134]: Updated as recommended.

**Review by anupama2501 — APPROVED** [review]

**Pull request merged**